.github/workflows/bpf-checks.yaml

```diff
@@ -7,22 +7,22 @@ on:
       - master
 
 jobs:
-    checkpatch:
-        name: checkpatch
-        runs-on: ubuntu-latest
-        steps:
-            - name: Checkout code
-              uses: actions/checkout@v2
-              with:
-                  fetch-depth: 0
-            - name: Run checkpatch.pl
-              run:
-                  make -C bpf checkpatch
-    coccicheck:
-        name: coccicheck
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/checkout@v2
-            - uses: docker://cilium/coccicheck:1.0
-              with:
-                entrypoint: ./contrib/coccinelle/check-cocci.sh
+  checkpatch:
```

Comment:
opps this one was done by my IDE file watcher. The identation was not consistent.

This is not related to scope of this PR, let me know if it's ok, otherwise I will revert.